internal/extproc/translator/openai_awsbedrock.go

```diff
@@ -95,26 +96,43 @@ func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) RequestBody(body router.R
 	}
 
 	mut := &extprocv3.BodyMutation_Body{}
-	if body, err := json.Marshal(bedrockReq); err != nil {
+	if b, err := json.Marshal(bedrockReq); err != nil {
 		return nil, nil, nil, fmt.Errorf("failed to marshal body: %w", err)
 	} else {
-		mut.Body = body
+		mut.Body = b
 	}
 	setContentLength(headerMutation, mut.Body)
 	return headerMutation, &extprocv3.BodyMutation{Mutation: mut}, override, nil
 }
 
+func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) openAIToBedrockInferenceConfiguration(
+	openAIReq *openai.ChatCompletionRequest, bedrockReq *awsbedrock.ConverseInput,
+) {
+	bedrockReq.InferenceConfig = &awsbedrock.InferenceConfiguration{}
+	if openAIReq.MaxTokens != nil {
+		bedrockReq.InferenceConfig.MaxTokens = openAIReq.MaxTokens
+	}
+	if openAIReq.Stop != nil {
+		bedrockReq.InferenceConfig.StopSequences = openAIReq.Stop
+	}
+	if openAIReq.Temperature != nil {
+		bedrockReq.InferenceConfig.Temperature = openAIReq.Temperature
+	}
+	if openAIReq.TopP != nil {
+		bedrockReq.InferenceConfig.TopP = openAIReq.TopP
+	}
```

Comment:
why do you need this ifs

Reply:
True this was my original code, ifs can be removed as they are all pointer types